messaging/src/main/java/org/axonframework/queryhandling/SimpleQueryBus.java

```diff
@@ -229,17 +236,17 @@ private boolean unsubscribe(String queryName,
                 .thenAccept(monoSink::success)
                 .exceptionally(t -> {
                     logger.error(format("An error happened while trying to report an initial result. Query: %s", query),
-                                 t);
+                            t);
                     monoSink.error(t.getCause());
                     return null;
                 }));
 
         UpdateHandlerRegistration<U> updateHandlerRegistration = queryUpdateEmitter
-                .registerUpdateHandler(query, backpressure, updateBufferSize);
+                .registerUpdateHandler(query, updateBufferSize);
 
         return new DefaultSubscriptionQueryResult<>(initialResult.getMono(),
-                                                    updateHandlerRegistration.getUpdates(),
-                                                    updateHandlerRegistration.getRegistration());
+                updateHandlerRegistration.getUpdates(),
```

Comment:
Similarly assuming that this indentation will be changed back if the style guide is being used.